modules/olm-operatorgroups-limitations.adoc

```diff
@@ -1,12 +1,13 @@
-[id="olm-operatorgroups-limitations"]
-= Limitations for multi-tenant Operator management
+// Module included in the following assemblies:
+//
+// * operators/understanding/olm/olm-understanding-operatorgroups.adoc
 
-{product-title} provides limited support for simultaneously installing different variations of an Operator on a cluster. Operators are control plane extensions. All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multi-tenant environment also have to share Operators.
+:_content-type: CONCEPT
+[id="olm-operatorgroups-limitations"]
+= Limitations for multitenant Operator management
 
-The Operator Lifecycle Manager (OLM) installs Operators multiple times in different namespaces. One constraint of this is that the Operator’s API versions must be the same.
+{product-title} provides limited support for simultaneously installing different variations of an Operator on a cluster. Operators are control plane extensions. All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multitenant environment must also share Operators.
```

Comment:
```suggestion
{product-title} provides limited support for simultaneously installing different versions of an Operator on the same cluster. Operators are control plane extensions due to their usage of `CustomResourceDefinitions` (which are global entities in Kubernetes). All tenants, or namespaces, share the same control plane of a cluster. Therefore, tenants in a multi-tenant cluster also share global `CustomResourceDefinitions` which limits the scenarios in which different instances of the same operator can be used in parallel on the same cluster. The scenarios include the following:

- operators in different versions that ship the exact same CRD definition (in case of versioned CRDs the exact same set of versions)
- operators in different versions that do not ship a CRD (and rather have their CRD available in a separate bundle on the OperatorHub)

All other scenarios are not supported, since the integrity of the cluster data cannot be guaranteed if there are multiple competing or overlapping `CustomResourceDefinitions` from different operator versions are to be reconciled on the same cluster.
```